CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 
+- Update README.md with current information
+
+### Changed
```

Comment:
Why two `Changed` ? Also, shouldn't this be in `Unreleased` ?

Reply:
Thanks for noticing. Of course that section was meant to go into `Unreleased`.